content/en/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -173,10 +183,161 @@ kube-scheduler has a default set of scheduling policies.
   that don't have Pods for the service already assigned there. The overall outcome is
   that the Service becomes more resilient to a single Node failure.
 
-- `CalculateAntiAffinityPriorityMap`: This policy helps implement
-  [pod anti-affinity](/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity).
-
-- `EqualPriorityMap`: Gives an equal weight of one to all nodes.
+- `EqualPriority`: Gives an equal weight of one to all nodes.
+
+- `EvenPodsSpreadPriority`: Implements preferred
+  [pod topology spread constraints](/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
+
+### Scheduling profiles
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+Scheduling profiles allow more flexibility when configuring a scheduler.
+In a profile, Plugins that implement scheduling behaviors can be enabled,
+disabled and reordered. A plugin implements one or more scheduling extension
+points, namely:
+
+1. **QueueSort**: These plugins provide an ordering function that is used to
+   sort pending Pods in the scheduling queue. Exactly one queue sort plugin
+   may be enabled at a time.
+1. **PreFilter**: These plugins are used to pre-process or check information
+   about a Pod or the cluster before filtering.
+1. **Filter**: These plugins are the equivalent of Predicates in a scheduling
+   Policy and are used to filter out nodes that can not run the Pod. Filters
+   are called in the configured order.
+1. **PreScore**: This is an informational extension point that can be used
+   for doing pre-scoring work.
+1. **Score**: These plugins provide a score to each node that has passed the
+   filtering phase. The scheduler will then select the node with the highest
+   weighted scores sum.
+1. **Reserve**: This is an informational extension point that notifies plugins
+   when resources have being reserved for a given Pod.
+1. **Permit**: These plugins can prevent or delay the binding of a Pod.
+1. **PreBind**: These plugins perform any work required before a Pod is bound.
+1. **Bind**: The plugins bind a Pod to a Node. Bind plugins are called in order
+   and once one has done the binding, the remaining plugins are skipped. At
+   least one bind plugin is required.
+1. **PostBind**: This is an informational extension point that is called after
+   a Pod has been bound.
+1. **UnReserve**: This is an informational extension point that is called if
+   a Pod is rejected after being reserved and put on hold by a Permit plugin.
```

Comment:
How do you feel about putting this level of detail in a page in the [Reference](https://kubernetes.io/docs/reference/) section?

Reply:
I assume you mean https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/ to be more specific. And that should include both `Scheduling Policies` and `Scheduling Profiles` sections. I think that would be better.

Reply:
A warning: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/ is [generated](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/) documentation.

So not that particular page, as it's not easy to add content to generated docs, but somewhere else in the reference section.

Reply:
In that case, see kubernetes/kubernetes#88919

Reply:
As long as the issue above is not resolved, I think adding another reference page would just add confusion about the intent of each file.

Reply:
I think it's OK to add a page with a URI like https://kubernetes.io/docs/reference/scheduling/plugins/ that lists all the plugins and what they do.

I'd see that as equivalent to the list of admission controllers in https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#what-does-each-admission-controller-do

Reply:
In that case, I think we can have:

- kubernetes.io/docs/reference/scheduling/policies
- kubernetes.io/docs/reference/scheduling/profiles

Which are linked from kube-scheduler-implementation (which I also plan to rename to `How a node is selected`)

Reply:
:ok_hand:

Reply:
Done